src/dead_code_elimination.cpp

```diff
@@ -50,7 +50,9 @@ void dead_code_elimination::apply(module& m) const
             break;
         // Skip instruction with empty shape as output unless its [dynamic, builtin, undefined,
         // identity, allocate]
```

Comment:
I think this comment should be updated as well

Reply:
Done.